docs/releases/2.0.txt

```diff
@@ -776,7 +776,8 @@ See :ref:`deprecated-features-1.10` for details on these changes.
 
 * The ``shell --plain`` option is removed.
 
-* The ``django.core.urlresolvers`` module is removed.
+* The ``django.core.urlresolvers`` module is removed in favor of its new
```

Comment:
This one is kind of mentioned above in the "see deprecated features in 1.10" link, but it took me some time to notice when upgrading and I thought inlining it here would be helpful since I think it will be very common.